app/views/comments/index.html.erb

```diff
@@ -98,14 +98,22 @@
         <% end %>
       </div>
     <% else %>
+    <% any_comments_hidden = false %>
       <% Comment.tree_for(@commentable).each do |comment, sub_comments| %>
+      <% any_comments_hidden = true if comment.hidden_by_commentable_user %>
         <% cache ["comment_root_#{user_signed_in?}", comment] do %>
           <%= tree_for(comment, sub_comments, @commentable) %>
         <% end %>
       <% end %>
     <% end %>
   </div>
   <center style="margin:30px 0px 22px;">
+  <% if any_comments_hidden %>
```

Comment:
This logic doesn't seem to be working for me

<img width="891" alt="Screen Shot 2019-12-02 at 5 17 46 PM" src="https://user-images.githubusercontent.com/3102842/69999867-b0a91400-1527-11ea-8dfa-e726b3ad3241.png">